controllers/registration_test.go

```diff
@@ -50,21 +50,22 @@ var _ = Describe("RegistrationController", func() {
 				}
 				body := strings.NewReader(form.Encode())
 				response := MakeRequest("POST", "/register", body)
-				flash := GetFlash(response.Result().Cookies())
+				flash := GetFlash(response.Cookies())
 
 				Expect(flash.Data["error"]).To(Equal("Name can not be empty"))
 			})
 
-			It("does NOT redirect to any new page", func() {
+			It("renders the same register page", func() {
```

Comment:
Not sure about the phrasing 🤔 How about:
```suggestion
			It("rerenders the registration page", func() {
```